coderd/provisionerdaemons.go

```diff
@@ -0,0 +1,81 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/coderd/database/db2sdk"
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/coderd/httpmw"
+	"github.com/coder/coder/v2/coderd/provisionerdserver"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+// @Summary Get provisioner daemons
+// @ID get-provisioner-daemons
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Enterprise
```

Comment:
Might need to rename / move this tag, not sure how much of a breaking change this would be?

Reply:
Good catch, this was not meant to be left in 👍🏻